src/main/java/command/Command.java

```diff
@@ -0,0 +1,100 @@
+package command;
+
+import task.TaskType;
+import time.Time;
+
+public class Command {
+
+    private CommandType cmdType;
+    private Time taskTimeInfo;
+    private TaskType taskType;
+    private String taskDescription;
+
+    /**
+     * Constructor
+     */
```

Comment:
I like how you gave a small description of each method throughout your code. Really helps the reader understand your code better.